app/views/steps/income/lost_job_in_custody/edit.html.erb

```diff
@@ -12,7 +12,7 @@
               <%= f.govuk_radio_button :lost_job_in_custody, choice.value do %>
                 <%= f.govuk_date_field :date_job_lost, maxlength_enabled: true,
                                        legend: {
-                                         text: t('.date_job_lost',),
+                                         text: t('.date_job_lost'),
                                          size: 's'
```

Comment:
For most dates, the designs are in fact wanting the bold label e.g. https://www.figma.com/file/thkvnDBbQbHlSqzt1TE3lk/Crime-Apply?type=design&node-id=5993%3A97363&mode=design&t=k8KnBJdFVjrDcotT-1

I think this might be the only instance of it not having a bold label